src/coreclr/src/vm/readytoruninfo.cpp

```diff
@@ -932,10 +934,7 @@ PCODE ReadyToRunInfo::GetEntryPoint(MethodDesc * pMD, PrepareCodeConfig* pConfig
     }
 
 done:
-    if (ETW_EVENT_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_DOTNET_Context, R2RGetEntryPoint))
-    {
-        ETW::MethodLog::GetR2RGetEntryPoint(pMD, pEntryPoint);
-    }
+    ETW::MethodLog::GetR2RGetEntryPoint(pMD, pEntryPoint);
```

Comment:
IIRC, the out-lined check here was added to fix a minor performance regression. The allocation of the huge stack-allocated buffer in the tracing helpers was showing up on the profile.

Reply:
Oh that's not desirable, I'll put it back in then for that helper and for the TypeLoadStop call, no need to have unnecessary performance regressions.